Sprint9-JS-Oject/tema1.js

```diff
@@ -0,0 +1,13 @@
+// Scrieți o funcție care creează un obiect pentru a păstra informații despre rețeta ta preferată
+
+function getObject(){
+    const reteta = {
+        titlu : "Pizza",
+        portii : 3,
+        incrediente : ["sunca", "cas", "ketchup"],
+    };
+
+    return reteta;
+}
+
+console.log(getObject());
```

Comment:
Pentru ca esti descurcaret in temele noastre, recomand sa adaugi si un input folositor pentru utilizator aici.

Optional, de asemenea, creeaza 2 retete.

Reply:
done